### PR TITLE
New: Edit footnotes cross check

### PR DESCRIPTION
**Pull request by KyleMacPherson**
Prior to this change, there was no cross-check or approval process for edit footnotes.

This commit adds this workflow process.

https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-398
https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-399

### CODE REVIEW

File: app/helpers/cross_check_helper.rb [hunk 1]
```diff
@@ -40,6 +40,8 @@ def table_partial(workbasket)
       "workbaskets/shared/steps/review_and_submit/edit_nomenclatures"
     elsif workbasket.type == 'create_footnote'
       "workbaskets/shared/steps/review_and_submit/create_footnote"
+    elsif workbasket.type == 'edit_footnote'
```
Comment (PanosParalakis) [inline app/helpers/cross_check_helper.rb]: You probably need to add the case in ` def cross_check_title` above as well

File: app/views/workbaskets/edit_footnote/steps/review_and_submit/_footnote.html.slim [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          | Footnote valid from
+          - footnote = Footnote.find(footnote_id: workbasket.settings.original_footnote_id, footnote_type_id: workbasket.settings.original_footnote_type_id)
+        th.name-column
+          a Footnote type
+        th.code-column
+          a Description
+        th.measures-column
+          a Associated measures
+        th.measures-column
+          a Associated goods codes
```
Comment (PanosParalakis) [inline app/views/workbaskets/edit_footnote/steps/review_and_submit/_footnote.html.slim]: Do we need those headers to be links?

File: app/views/workbaskets/edit_footnote/workflow_screens_parts/notifications/_block.html.slim [hunk 3]
```diff
@@ -0,0 +1,7 @@
+- extra_alert_css = workbasket_status_in_error_level? ? 'error' : 'info'
+
+.alert.create-measures-message-block class="alert--#{extra_alert_css}"
+  svg.icon class="icon--#{extra_alert_css}" version="1.1" viewbox=("0 0 24 28") xmlns="http://www.w3.org/2000/svg"
```
Comment (PanosParalakis) [inline app/views/workbaskets/edit_footnote/workflow_screens_parts/notifications/_block.html.slim]: You can probably use GDS notice element here.  See: https://govuk-elements.herokuapp.com/icons-images/ or https://govuk-elements.herokuapp.com/typography/#typography-warning-text

File: app/helpers/cross_check_helper.rb (hunk 1 above)
Comment (KyleMacPherson) [reply app/helpers/cross_check_helper.rb]: @PanosParalakis nice, thanks man

**Pull request merged**